src/seam-connect/client.ts

```diff
@@ -201,7 +206,7 @@ const getAuthHeaders = ({
       )
       return { "client-session-token": apiKey }
     }
-    if (!apiKey.startsWith("seam_at") && workspaceId)
+    if (isValueUsedForAPIKeyAuthentication(apiKey) && workspaceId)
```

Comment:
Can we rename this isValueUsedForBearerAuthentication? Session token authentication is different from api key authentication

Reply:
The variable names are not great in this function...

Reply:
Yep, I renamed it. I reversed the logic though, since this if statement throws an error if it's not supposed to be used for bearer authentication